src/sql/azdata.proposed.d.ts

```diff
@@ -598,6 +600,11 @@ declare module 'azdata' {
 		 * The type to display the text as - used to determine the color of the text. Default is Normal.
 		 */
 		textType?: TextType;
+		/**
+		 * Provide text for HTML list. Note: By default the lists will be unordered. To change the list type use textType attribute.
+		 */
+		values?: string[];
```

Comment:
how about add this type to the existing value property instead of introducing a new property